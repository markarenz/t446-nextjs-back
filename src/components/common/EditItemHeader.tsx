import styles from '../../styles/modules/EditItemHeader.module.scss';
type Props = {
  title: string;
};
const EditItemHeader: React.FC<Props> = ({ title }) => {
  return (
    <div className={styles.pageHeader}>
      <h2 className="pageTitle">{title}</h2>
    </div>
  );
};

export default EditItemHeader;
