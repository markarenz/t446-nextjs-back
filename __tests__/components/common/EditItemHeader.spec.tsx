import React from 'react';
import { render, screen, fireEvent, waitFor } from '@testing-library/react';
import EditItemHeader from '../../../src/components/common/EditItemHeader';

const defaultProps = {
  title: 'Test Title'
};

describe('EditItemHeader', () => {
  it('renders header', async () => {
    render(<EditItemHeader {...defaultProps} />);
    const element = await screen.findByText('Test Title');
    expect(element).toBeInTheDocument();
  });
});
