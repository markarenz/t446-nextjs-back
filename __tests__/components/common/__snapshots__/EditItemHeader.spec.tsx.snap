// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditItemHeader renders header 1`] = `
<h2
  class="pageTitle"
>
  Test Title
</h2>
`;
